app/styles/modules/_settings.scss

```diff
@@ -807,3 +807,11 @@ section.modal-panel {
     }
   }
 }
+
+#unblock_code {
```

Comment:
This is probably not the best place to put this style. I think putting it in [_input-row.scss](https://github.com/mozilla/fxa-content-server/blob/a5c28820a8f2ce273de5218ff7d95bc9333c9b21/app/styles/modules/_input-row.scss#L25) makes more sense.

Also, you can reference class instead of the id to be more generic. Ex.

```
      &.unblock-code {
          text-transform: uppercase;
          &::placeholder {
              text-transform: none;
          }
      }
```